argilla-frontend/components/features/annotation/container/fields/custom-field/CustomField.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <div class="custom_field_component" :key="content">
+    <div class="title-area --body2">
+      <span class="custom_field_component__title-content" v-text="title" />
+      <BaseActionTooltip
+        class="custom_field_component__tooltip"
+        tooltip="Copied"
+        tooltip-position="left"
+      >
+        <BaseButton
+          title="Copy to clipboard"
+          class="custom_field_component__copy-button"
+          @click.prevent="$copyToClipboard(content)"
+        >
+          <svgicon color="#acacac" name="copy" width="18" height="18" />
+        </BaseButton>
+      </BaseActionTooltip>
+    </div>
+    <div :id="`fields-content-${name}`" class="content-area --body1">
+      <Sandbox :content="template" />
+    </div>
+  </div>
+</template>
+<script>
+/* eslint-disable */
+const STYLES = `
+<script>
+if (parent) {
+  const currentHead = document.getElementsByTagName("head")[0];
+  const styles = parent.document.getElementsByTagName("style");
+  for (const style of styles) {
+    currentHead.appendChild(style.cloneNode(true));
+  }
+}
+<\/script>`;
+
+const BASIC_TEMPLATE = `
+${STYLES}
```

Comment:
@damianpumar @burtenshaw I think it would be more straightforward to just use `record` as naming instead of `record_object`. WDYT?

Reply:
Sounds good to me.

Reply:
Yes,! agree. I change it now!

Thanks